### PR TITLE
[MRG] Update ChangeLog to release 0.8.0

### PR DESCRIPTION
**Pull request by betatim**
Originally thought we hadn't changed much so the next release would be 0.7.1.

However while collecting things for the change log I changed my mind. There are quite a few changes, most in the "making things more stable and reliable and conformant" department. Should we call it 0.8.0?

Please make suggestions about the wording for the change log via the GitHub UI. Not quite sure what tense/form they should take.

### CODE REVIEW

**Comment by minrk** [comment]
sounds good to me!

**Comment by yuvipanda** [comment]
@minrk @betatim I'd like https://github.com/jupyter/repo2docker/pull/539 to be in this...

**Comment by betatim** [comment]
The "major" version bump in #539: do you mean major as in 0.7.0 -> 1.0.0 or 0.7.0 -> 0.8.0?

If we do follow sem-ver then I think as long as it is 0.x.y we could go with 0.7.0 -> 0.8.0 as a "major" bump. It seems the rules suggest that "as long as the major is 0 anything might change at any moment".

**Comment by betatim** [comment]
Looking at the change log I think we should go ahead and call this 0.8.0.

**Comment by betatim** [comment]
Let's release this tomorrow before the meeting or just after as the change log is already starting to miss things.

**Comment by betatim** [comment]
Merging this. If we merge more PRs that need to be in the change log please add a line as part of the PR.

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -3,21 +3,50 @@ Changelog
 =========
 
 
-Upcoming release
-================
+Version 0.8.0
+=============
 
-Release date: TBD
+Release date: 2019-02-16
```
Comment (betatim) [inline docs/source/changelog.rst]: Need to update the date before cutting the release :-/

**Comment by yuvipanda** [comment]
Thank you very much for waiting, @betatim!

**Pull request merged**